docs/src/docs/asciidoc/user/serializers-copiers.adoc

```diff
@@ -145,6 +151,14 @@ will be sanctioned with an exception upon cache initialization.
 
 Registering a serializer that lacks such constructor at the cache manager level will prevent it from being chosen for persistent caches.
 
+NOTE: The constructor with the signature `(ClassLoader classLoader, FileBasedPersistenceContext persistenceContext)`
```

Comment:
I would make the note much shorter sa we do not want too much focus on this.

Something like:
> The constructor with the signature `(ClassLoader classLoader, FileBasedPersistenceContext persistenceContext)` that existed in v3.0 is still supported to respect backward compatibility but its usage is limited to disk based caches.